src/devices/SenseHat/samples/SenseHat.Sample.cs

```diff
@@ -40,12 +45,29 @@ public static void Main(string[] args)
                     sh.Fill(n % 2 == 0 ? Color.DarkBlue : Color.DarkRed);
                     sh.SetPixel(x, y, Color.Yellow);
 
-                    Console.WriteLine($"Temperature: Sensor1: {sh.Temperature.Celsius} °C   Sensor2: {sh.Temperature2.Celsius} °C");
-                    Console.WriteLine($"Humidity: {sh.Humidity} %rH");
-                    Console.WriteLine($"Pressure: {sh.Pressure} hPa");
-                    Console.WriteLine($"Acceleration: {sh.Acceleration} g");
-                    Console.WriteLine($"Angular rate: {sh.AngularRate} DPS");
-                    Console.WriteLine($"Magnetic induction: {sh.MagneticInduction} gauss");
+                    var tempValue = sh.Temperature;
```

Comment:
I'd revert most of the changes from here. This sample is showcasing SenseHat mainly and I feel like most of those weather helpers here are obscuring the usage a bit

Reply:
Maybe we can separate them instead of eliminate them?

Reply:
I'd make it as crisp as possible so rather keeping just 1 so that people will explore the type if they need it but no more in case they are not interested to not distract from what we are trying to show

Reply:
Done (pending commit)